insert.go

```diff
@@ -120,6 +130,29 @@ func (ib *InsertBuilder) BuildWithFlavor(flavor Flavor, initialArg ...interface{
 	buf := newStringBuilder()
 	ib.injection.WriteTo(buf, insertMarkerInit)
 
+	if ib.sbHolder != "" && ib.args.Flavor == Oracle {
```

Comment:
INSERT INTO ... SELECT statement is not an Oracle-only SQL syntax. AFAIK, at least MySQL and MS SQLServer support this statement. So I don't think we need to check flavor here.

Reply:
You are right, I was confused by the different approach for the Oracle here. Updated.